trojsten/submit/templates/trojsten/submit/protocol.html

```diff
@@ -34,7 +34,9 @@ <h4>Výstup z kompilátora</h4>
                     {% if test.showDetails %}
                 <tr>
                 <td colspan="3"  style="padding:0">
-                        <div  class="collapse" id="collapse{{ forloop.counter0 }}"> {{ test.details|linebreaksbr }}</div>
+                    <div class="collapse" id="collapse{{ forloop.counter0 }}">
+                        <pre>{{ test.details|linebreaksbr }}</pre>
```

Comment:
Ten `linebreaksbr` filter tam chce zostať?

Reply:
Máš pravdu, technicky tam asi byť nemusí, keďže je to v pre.